### PR TITLE
Hotfix for v4.3.1 - Revert - Adding ability to allow worker to send Function load response

### PR DESCRIPTION
**Pull request by surgupta-msft**
Commit id - 97f14a186863ed4796b2174278c19758962e1893

PR - https://github.com/Azure/azure-functions-host/pull/8346

### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### Additional information

Additional PR information

### CODE REVIEW

**Review by liliankasem — APPROVED** [review]

**Review by pragnagopa — APPROVED** [review]

File: build/common.props [hunk 1]
```diff
@@ -4,8 +4,8 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
-    <MinorVersion>2</MinorVersion>
```
Comment (fabiocav) [inline build/common.props]: For a patch, this should be targeting the same minor with a different patch. Is there a reason this is jumping here?

Comment (surgupta-msft) [reply build/common.props]: Tagging Jameson here. @VpOfEngineering

Comment (VpOfEngineering) [reply build/common.props]: Yes, it was a merge conflict (remote was 4.2.1, local was 4.3.1) and I wrongly selected 4.2.1 for this file. We could re-do the PR with that selection switched

**Review by fabiocav — APPROVED** [review]
